README.md

```diff
@@ -13,10 +13,10 @@ A proposed [WebAssembly System Interface](https://github.com/WebAssembly/WASI) A
 - [Jiaxiao Zhou](https://github.com/Mossaka)
 - [Taylor Thomas](https://github.com/thomastaylor312)
 
-### Phase 4 Advancement Criteria
+### Portability Criteria
 
-`wasi-runtime-config` should have at least two implementations (i.e., from service providers, and or
-cloud providers), and, at the very minimum, pass the testsuite for Windows, Linux, and MacOS.
+`wasi-runtime-config` should have at least two implementations in runtimes and/or service
+providers/platforms and pass the testsuite for Windows, Linux, and MacOS.
```

Comment:
```suggestion
`wasi-runtime-config` must have at least two implementations in terms of relevant mainstream config providers or platforms such as HashiCorp Vault, Redis, AWS Secrets Manager, GCP Secret Manager, Azure Key Vault etc.. The implementations must be able to run on Linux, MacOS, and Windows.
```

Reply:
Are there any other examples that don't just involve secrets? I figured we'd be adding a secrets interface in the future as part of runtime config so I didn't want to mislead people

Reply:
emm that's a good question.
 
Azure App Configuration, Apache Zookeeper and perhaps etcd?

Reply:
I will @devigned for more inputs here

Reply:
The interesting part here is that I feel like a bunch of platforms will likely implement this. We already have it in wasmcloud (provided by the wasmcloud host), things like Spin will also have an integration, and runwasi will likely have some sort of wasi-virt adapter that can convert env vars to config

Reply:
Ok, I pushed a change up. Let me know what you think